cfgov/jinja2/v1/about-us/careers/_single.html

```diff
@@ -98,39 +98,38 @@ <h2 class="u-mb0" id="interested">Interested in Applying?</h2>
                 } ) }}
             </div>
         </div>
-        <div class="block">
-            <h3>Before you apply</h3>
-            <p>
-                {# TODO: Replace with real content. #}
-            </p>
-            <ul class="list list__links">
-                <li class="list_item">
-                    <a class="jump-link"
-                       href="/about-us/careers/working-at-cfpb/">
-                        Learn about working @ CFPB
-                    </a>
-                </li>
-                <li class="list_item">
-                    <a class="jump-link"
-                       href="/about-us/careers/application-process/">
-                        Learn about the application process
-                    </a>
-                </li>
-            </ul>
-        </div>
-        {% for job_applicant_type in career.jobapplicanttype_set.all() %}
-        <div class="block block__bg block__border">
-            <h4>{{ job_applicant_type.application_type.applicant_type }}</h4>
-            <p>{{ job_applicant_type.application_type.description | striptags }}</p>
-
-            <p><a class="btn" href="{{ job_applicant_type.usajobs_url }}">Apply now</a></p>
-
-            <p>
-                You are about to leave consumerfinance.gov. To submit the application, you must go to USAJobs.gov.
-            </p>
-        </div>
-        {% endfor %}
+        <h3>Before you apply</h3>
+        {# TODO: Replace with real content. #}
+        {#<p>
```

Comment:
For the time being the "real content" is those links to Working @ CFPB and the Application Process page, so I think the `p` can be removed.